.github/workflows/test-all-warehouses.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        dbt-version: ["1.3.0", null]
```

Comment:
This is a bit annoying when using `workflow_dispatch` because it'll create jobs that'll immediately fail like [this](https://github.com/elementary-data/elementary/actions/runs/5923924227).